lib/run.js

```diff
@@ -11,12 +11,23 @@ const reInterval = require('reinterval')
 const histAsObj = histUtil.histAsObj
 const addPercentiles = histUtil.addPercentiles
 
-function run (opts, cb) {
+function run (opts, cb, tracker) {
```

Comment:
Can you rename this `_run` and have the wrapper be `run` instead? I would like to avoid passing in the extra `tracker` argument, it could lead to side effects.

Reply:
fixed